app/src/org/odk/collect/android/views/ODKView.java

```diff
@@ -434,10 +426,6 @@ public void removeWidget(View v){
      * @return Index of question's view in mView.
      */
     private int getViewIndex(int questionIndex) {
-        // Account for progress bar
-        if (mProgressEnabled) {
```

Comment:
@phillipm Are you sure this code isn't relevant anymore though? I don't know this part of the code, just seems like a weird thing to be removing